source/code/plugins/in_npmd_server.rb

```diff
@@ -334,6 +343,28 @@ def check_and_update_binaries
             File.unlink(_x64BinPath) if File.exist?(_x64BinPath)
         end
 
+        def check_agent_capability
+            return if !File.exist?(@location_agent_binary) or @skip_capability_check
+
+            # Checking if capabilties are proper else delete binary and version file
+            _getCapResult = `getcap #{@location_agent_binary}`
+            if !_getCapResult.include?(NPMD_AGENT_CAPABILITY)
+                # Delete agent binary
+                File.unlink(@location_agent_binary)
+
+                # Delete version file(s)
+                _fileLocPostFix = "#{NPMD_VERSION_FILE_DIR}/#{NPMD_VERSION_FILE_NAME}"
```

Comment:
Removing the same.

Also adding the case when NPMD binary is absent but version files are present